controllers/apps/v2beta1/add_bootstrap_resource_suite_test.go

```
@@ -22,6 +22,7 @@
 
 	BeforeEach(func() {
 		a = &addBootstrap{emqxReconciler}
+		a.LoadEMQXConf(instance)
```

golangci-lint (1.64.8)

Error return value of `a.LoadEMQXConf` is not checked

(errcheck)